commands/show-current-unix-timestamp/command.toml

```diff
@@ -0,0 +1,16 @@
+name = "Show Current UNIX Timestamp"
+description = "Quickly see or copy the current UNIX timestamp"
+endpointURL = "https://slapcommands.vercel.app/api/utils/unix-timestamp"
+sourceCodeURL = "https://github.com/AnishDe12020/slapcommands/blob/main/src/shared/utils/unix-timestamp.js"
```

Comment:
@AnishDe12020 

A few suggestions here:

- Change the option to "Copy Timestamp"
- Move the actual timestamp to subtitle, so it will be like `{timestamp} • {full date}`

![CleanShot 2022-04-05 at 11 32 09](https://user-images.githubusercontent.com/57078134/161735379-c71d7939-439e-4062-9e78-605b9faf616e.png)

Reply:
Yeah makes sense, I have made the change and pushed.